src/test/java/io/openliberty/tools/intellij/it/SingleModMPProjectTestCommon.java

```diff
@@ -1007,6 +1029,59 @@ public static void deleteDirectoryIfExists(String dirPath) {
         }
     }
 
+    /**
+     * Clean project.
+     */
+    public void stopTerminal() {
+
+        Keyboard keyboard = new Keyboard(remoteRobot);
+        ProjectFrameFixture projectFrame = remoteRobot.find(ProjectFrameFixture.class, Duration.ofSeconds(10));
+        ComponentFixture terminal = remoteRobot.find(ComponentFixture.class, byXpath("//div[@class='JBTerminalPanel']"), Duration.ofSeconds(10));
+
+        terminal.rightClick();
+        ComponentFixture openFixtureNewTab = projectFrame.getActionMenuItem("New Tab");
+        openFixtureNewTab.click(new Point());
+
+        // Perform Stop Action
+        if (getBuildCategory() == BuildType.MAVEN_TYPE) {
+            keyboard.enterText("./mvnw liberty:stop");
+
+        } else if (getBuildCategory() == BuildType.GRADLE_TYPE) {
+            keyboard.enterText("./gradlew libertyStop");
+        }
+        keyboard.enter();
```

Comment:
I think it is worth adding an ELSE where you print a message in the log saying an invalid build type was specified. ` TestUtils.printTrace("Invalid ...");` Or maybe even an Assertion to stop all testing to make sure the tests are always correctly specified. See if Trevor or Michael have an opinion.

Reply:
I have added a printTrace for now. If needed we can add Assertion by discussing further with the team.

Reply:
If there is a WARNING level I think that would be better.